.github/workflows/scriptTests.yml

```diff
@@ -71,7 +76,7 @@ jobs:
         run: |
           sh build/build.sh
           ./jf${{ matrix.suite.osSuffix }} --version
-        if: contains( matrix.suite.os, 'windows')
+        if: ${{ !contains( matrix.suite.os, 'windows') }}
```

Comment:
not sure that !contains is supported 
you can take my changes here - https://github.com/jfrog/jfrog-cli/pull/2448/files